src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const logo = document.getElementById('logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error());
+  }, 3000);
+});
+
+const onSuccess = () => {
+  const message = document.createElement('div');
+
+  message.className = 'message';
+  message.textContent = 'Promise was resolved!';
+  document.body.appendChild(message);
+};
+
+const onError = () => {
+  const error = document.createElement('div');
+
+  error.className = 'message error-message';
+  error.textContent = 'Promise was rejected!';
+  document.body.appendChild(error);
+};
+
+promise1.then(onSuccess).catch(onError);
+promise2.then(onSuccess).catch(onError);
```

Comment:
Both promises are handled separately, which means `onSuccess` or `onError` will be called twice: once for each promise. If you intend to handle them together, consider using `Promise.race` or `Promise.all` to manage the promises collectively. This way, you can handle the first promise that resolves or rejects, or wait for both to settle, depending on your requirements.